transforms/transform.config

```diff
@@ -14,4 +14,4 @@ TRANSFORM_NAME=data-prep-kit-transforms
 #
 # If you change the versions numbers, be sure to run "make set-versions" to 
 # update version numbers across the transform (e.g., pyproject.toml).
-TRANSFORMS_PKG_VERSION=$(DPK_VERSION)
+TRANSFORMS_PKG_VERSION=0.2.2.dev2
```

Comment:
Does `make set-versions` update this or is this a hand-edit?  If a hand-edit, wondering why not just stay with $(DPK_VERSION) or some other globally defined version that is used by `make set-versions`.

Reply:
@daw3rd this follows a similar pattern to the other transforms where each transform can define its own revision tag. Our starting point was DPK_VERSION for all the transforms but expect to diverge going forward